test/util_test.go

```diff
@@ -152,10 +159,19 @@ func (ks *kineServer) ReportMetrics(b *testing.B) {
 	b.ReportMetric(float64(sqliteMetrics.PageCacheWrites)/float64(b.N), "page-writes/op")
 	b.ReportMetric(float64(sqliteMetrics.TransactionReadTime)/float64(time.Second)/float64(b.N), "sec-reading/op")
 	b.ReportMetric(float64(sqliteMetrics.TransactionWriteTime)/float64(time.Second)/float64(b.N), "sec-writing/op")
+
+	if ks.dqliteListener != nil {
+		dqliteMetrics := ks.dqliteListener.Metrics()
+		b.ReportMetric(float64(dqliteMetrics.BytesRead)/float64(b.N), "network-reads/op")
+		b.ReportMetric(float64(dqliteMetrics.BytesWritten)/float64(b.N), "network-writes/op")
```

Comment:
I think the metric names are slightly off here, it looks like the number of times the network is used, but the value is in terms of bytes. Perhaps these could be renamed to something like these?
- `network-bytes-read/op`
- `network-bytes-written/op`